core/chains/evm/config/toml/defaults/Avalanche_Fuji.toml

```diff
@@ -1,5 +1,6 @@
 ChainID = '43113'
 FinalityDepth = 1
+FinalityTagEnabled = false
```

Comment:
let's add a comment why we set it to false even though technically it supports it, same for other Avax